releases/evm/2025-02-XRPL-EVM-GMP-v6.0.4.md

```diff
@@ -12,43 +12,43 @@
 | **Testnet**          | -                     | TBD      |
 | **Mainnet**          | -                     | TBD      |
 
-- [Releases](https://github.com/axelarnetwork/axelar-cgp-solidity/tree/v6.4.0)
+- [Releases](https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/releases/tag/v6.0.4)
 
 ## Background
 
 Changes in the release:
 
-This is the v6.4.0 deployment of EVM compatible Amplifier Gateway contracts for Ripple Sidechain.
+This is the v6.0.4 deployment of EVM compatible Amplifier Gateway contracts for XRPL Sidechain.
 
 ## Deployment
 
-Create a `.env` config. `CHAIN` should be set to `xrplevm` for all production networks: `mainnet`, `testnet`, `stagenet` and `devnet-amplifier`.
+Create a `.env` config. `CHAINS` should be set to `$CHAINS` for all production networks: `mainnet`, `testnet`, `stagenet` and `devnet-amplifier`.
```

Comment:
`CHAINS` is not defined in this doc. Describe the values here since it's not clear to someone that the other doc should be checked first